vc-verifier/kotlin/vcverifier/src/test/java/io/mosip/vercred/vcverifier/credentialverifier/types/LdpVcCredentialVerifierTest.kt

```diff
@@ -0,0 +1,42 @@
+package io.mosip.vercred.vcverifier.credentialverifier.types
+
+import android.util.Log
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mosip.vercred.vcverifier.CredentialsVerifier
+import org.junit.After
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+class LdpVcCredentialVerifierTest {
+    @Before
+    fun before() {
+        mockkStatic(Log::class)
+        every { Log.e(any(), any()) } returns 0
+        every { Log.e(any(), any(), any()) } returns 0
+        every { Log.i(any(), any()) } returns 0
+        every { Log.w(any(), any<String>()) } returns 0
+    }
+
+    @After
+    fun after() {
+        clearAllMocks()
+    }
+
+    @Test(timeout = 10000)
+    fun `should return true for valid credential`() {
+        val validVC =
```

Comment:
Is it fine to push valid VC ?

Reply:
yes, have already discussed it with Anusha and Vishwa.
These is qa env so valid test VC